packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -407,3 +423,47 @@ int stackFrameIdCompare(String a, String b) {
     throw error;
   }
 }
+
+class CpuProfileStore {
+  final _store = <TimeRange, CpuProfileData>{};
+
+  CpuProfileData storedProfile(TimeRange time) {
```

Comment:
nit: I would remove the worst store from most of these names. It just makes it a little confusing.

Reply:
renamed to `profileForTimeRange` and `addProfile`